cocotb/share/lib/vpi/VpiImpl.cpp

```diff
@@ -241,7 +241,7 @@ GpiObjHdl* VpiImpl::create_gpi_obj_from_handle(vpiHandle new_hdl,
 
     new_obj->initialise(name, fq_name);
 
-    LOG_DEBUG("VPI: Created object with type was %s(%d)",
+    LOG_DEBUG("VPI: Created GPI object from type %s(%d)",
```

Comment:
```suggestion
    LOG_DEBUG("VPI: Created GPI object of type %s(%d)",
```
?

Reply:
I don't think that would be correct as the ```type %s(%d)``` will print the VPI types, not the GPI types.

Reply:
I'm not sure that's wrong, GPI is an abstraction around VPI and friends so making a GPI object of a concrete VPI type makes sense, at least to me. I'm not that worried if you want to leave it, it's a debug log, it's not really meant for an average end user.